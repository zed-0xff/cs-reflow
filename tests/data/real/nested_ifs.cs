	public unsafe static string a6b9c1c081314e13bfac93c6ac01a767(object object_0as)
	{
		string a7adf881292040d386ae7ea0d9c98fce = Strings_a392.a7adf881292040d386ae7ea0d9c98fce;
		int num = 206;
		if (!String.op_Equality((string)object_0as, a7adf881292040d386ae7ea0d9c98fce))
		{
			goto IL_001a;
		}
		goto IL_004f;
		IL_001a:
		num = (num & -5) | 0x111;
		int num2;
		int num3;
		if (!String.op_Equality((string)object_0as, Strings_a392.a5ba8d381fd246b9a83132de88ccff0d))
		{
			num2 = -1842999025;
			num3 = -1842999025;
		}
		else
		{
			num2 = 1294805223;
			num3 = 1294805223;
		}
		goto IL_009b;
		IL_00c8:
		num2 = -1353270567;
		int num4 = -1353270567;
		goto IL_009b;
		IL_015c:
		num2 = 940860412;
		num4 = 940860412;
		goto IL_009b;
		IL_009b:
		int num5 = num2;
		num = (num & -5) | 0x111;
		int num6 = -712264766;
		goto IL_005f;
		IL_005f:
		uint num9;
		while (true)
		{
			int num7 = num6 ^ -853551246;
			num = (num & -5) | 0x111;
			uint num8 = (uint)num7;
			switch ((uint)num7 % 6u) // 4
			{
			case 0u:
				break;
			case 5u:
				goto IL_0096;
			case 3u:
				goto IL_00ae;
			case 4u:
				goto IL_00dc;
			default:
				goto IL_015c;
			case 2u:
				goto IL_024c;
			}
			break;
			IL_00dc:
			switch ((num9 = (uint)(-(~(~(-1253292810 - num5) + 2101500157)))) % 16) // 3
			{
			case 8u:
				break;
			case 5u:
				goto IL_0096;
			case 9u:
				goto IL_00ae;
			case 15u:
				goto IL_0138;
			default:
				goto IL_014a;
			case 14u:
				goto IL_016b;
			case 3u:
				goto IL_018c;
			case 4u:
				goto IL_01c2;
			case 12u:
				return Strings_a392.a4115adbc2404fd7b2d6a876a5e57357;
			case 7u:
				return Strings_a392.eb34feb6eeba42eca3cdd9a44e583316;
			case 13u:
				return Strings_a392.bb2a10ded574403e98317a61127245ee;
			case 11u:
				return Strings_a392.ac5923d2e8494cc0b3ab270d681f6e5c;
			case 2u:
				goto IL_01ec;
			case 0u:
				goto IL_01fe;
			case 6u:
				goto IL_0210;
			case 10u:
				return Strings_a392.c110efd81e7b4317a4a49f8c9d041c90;
			case 1u:
				goto IL_024c;
			}
			goto IL_001a;
			IL_024c:
			return (string)object_0as;
			IL_014a:
			num6 = ((int)num8 * -945321588) ^ -1005787634;
			continue;
			IL_00ae:
			if (!String.op_Equality((string)object_0as, Strings_a392.d30c8335f6e34dedb062401eb494970c))
			{
				num6 = -1992159495;
				continue;
			}
			goto IL_00c8;
		}
		goto IL_004f;
		IL_004f:
		num = (num & -5) | 0x111;
		num6 = -1990032839;
		goto IL_005f;
		IL_0096:
		num2 = 1617817432;
		goto IL_009b;
		IL_01fe:
		num2 = ((int)num9 * -1439152985) ^ 0x7ABCD910;
		goto IL_009b;
		IL_016b:
		num2 = (int)((num9 * 659937581) ^ 0x6FED2FC5);
		goto IL_009b;
		IL_01ec:
		num2 = (int)(num9 * 1310690800) ^ -1836432971;
		goto IL_009b;
		IL_018c:
		int num10;
		if (!String.op_Equality((string)object_0as, Strings_a392.ad0552d5a15e4e3ba3f88d9c603d48ae))
		{
			num2 = -1742904678;
			num10 = -1742904678;
		}
		else
		{
			num2 = 1260500939;
			num10 = 1260500939;
		}
		goto IL_009b;
		IL_0138:
		num2 = ((int)num9 * -856791433) ^ 0x3EEBC23D;
		goto IL_009b;
		IL_0210:
		int num11;
		if (!String.op_Equality((string)object_0as, Strings_a392.b5643a2021fa4247a84d55d813b16724))
		{
			num2 = 1068948706;
			num11 = 1068948706;
		}
		else
		{
			num2 = -1798756451;
			num11 = -1798756451;
		}
		goto IL_009b;
		IL_01c2:
		num2 = (int)(num9 * 1712772123) ^ -1576169574;
		goto IL_009b;
	}
