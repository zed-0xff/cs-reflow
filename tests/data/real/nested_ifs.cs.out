	public unsafe static string a6b9c1c081314e13bfac93c6ac01a767(object object_0as)
	{
	    string a7adf881292040d386ae7ea0d9c98fce = Strings_a392.a7adf881292040d386ae7ea0d9c98fce;
	    if (String.op_Equality((string)object_0as, a7adf881292040d386ae7ea0d9c98fce))
	    {
	        return Strings_a392.ac5923d2e8494cc0b3ab270d681f6e5c;
	    }
	
	    if (String.op_Equality((string)object_0as, Strings_a392.a5ba8d381fd246b9a83132de88ccff0d))
	    {
	        return Strings_a392.c110efd81e7b4317a4a49f8c9d041c90;
	    }
	
	    if (String.op_Equality((string)object_0as, Strings_a392.b5643a2021fa4247a84d55d813b16724))
	    {
	        return Strings_a392.bb2a10ded574403e98317a61127245ee;
	    }
	
	    if (String.op_Equality((string)object_0as, Strings_a392.d30c8335f6e34dedb062401eb494970c))
	    {
	        return Strings_a392.eb34feb6eeba42eca3cdd9a44e583316;
	    }
	
	    if (String.op_Equality((string)object_0as, Strings_a392.ad0552d5a15e4e3ba3f88d9c603d48ae))
	    {
	        return Strings_a392.a4115adbc2404fd7b2d6a876a5e57357;
	    }
	
	    return (string)object_0as;
	}
